scripts/register-cluster-with-dynatrace.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env sh
```

Comment:
use bash? all the other scripts use it
```suggestion
#!/usr/bin/env bash
```